translator/translate/otel/common/common.go

```diff
@@ -18,6 +18,7 @@ import (
 
 const (
 	AgentKey                           = "agent"
+	Debug                              = "debug"
```

Comment:
nit: For consistency.
```suggestion
	DebugKey                              = "debug"
```